scene/3d/area_3d.cpp

```diff
@@ -473,19 +473,19 @@ bool Area3D::is_monitoring() const {
 }
 
 TypedArray<Node3D> Area3D::get_overlapping_bodies() const {
-	ERR_FAIL_COND_V(!monitoring, Array());
-	Array ret;
+	ERR_FAIL_COND_V_MSG(!monitoring, Array(), "Can't find overlapping bodies when monitoring is off.");
+	TypedArray<Area3D> ret;
```

Comment:
This should be `TypedArray<Node3D>`.

Reply:
Actually it can be even simpler:
```C++
TypedArray<Area3D> ret;
ERR_FAIL_COND_V_MSG(!monitoring, ret, "Can't find overlapping bodies when monitoring is off.");
```